memcached.c

```diff
@@ -3321,7 +3321,8 @@ static void drive_machine(conn *c) {
                 if (q->stack_ctx != NULL) {
                     io_queue_cb_t *qcb = thread_io_queue_get(c->thread, q->type);
                     qcb->submit_cb(q);
-                    c->io_queues_submitted++;
+                    if (q->count != 0)
```

Comment:
took me a while to remember why this got added, need to add a comment:
- for await_background case, it short-circuits and returns the IO without increasing `q->count`
- so if the only requests submitted here are await_background, we need to not suspend the client as there's nothing to wait for

Reply:
actually nevermind, that was extraneous from a prior attempt at allowing zero queues. I've removed the line and run some benchmarks and it seems fine.